packages/dds/tree/src/test/feature-libraries/optional-field/optionalField.spec.ts

```diff
@@ -76,7 +76,7 @@ const change1 = tagChangeInline(
 
 const change2Tag = mintRevisionTag();
 const change2: TaggedChange<OptionalChangeset> = tagChangeInline(
```

Comment:
`tagChangeInline` does the same kind of work as `this.changeFamily.rebaser.changeRevision(change, revision)` in `Branch.apply` (although it also warps the input in a `TaggedChange`.
Since we're making changesets have the revision tags from the start, calls to `tagChangeInline` should no longer be necessary. Instead they should be replaced by calls to `tagChange` (or `tagRollbackInverse`).

In order to keep the test data unchanged, you do need to make sure that the revision tag used in the construction of the changeset is the same as the revision tag that is currenlty passed to `tagChangeInline`.